### ISSUE TITLE
Failing drag and drop operation from "Bitvise SSH client"

### ISSUE THREAD

**Issue by WhiteChairFromIkea** [double-commander/doublecmd#14]
# Issue
Drag and drop operation from SFTP client file explorer to DC file panel drops kind of GUID file, instead of real file:

see: https://youtu.be/9mAuucn2sRQ

# Steps to reproduce
1. Install https://www.bitvise.com/ssh-client-download
2. Login to test.rebex.net:22, login information: demo/password;
3. Drag and drop file from SFTP server to regular explorer ("MS explorer.exe", or "free commander");
4. File transfer is successful, file is created on local drive with correct filename;
5. Repeat step 3 with DC. Something is transfered from SFTP to local PC, but filename is not correct.

```
Double Commander
Version: 0.9.10 beta
Revision: 9640M
Build date: 2021/01/10
Lazarus: 2.0.8.0
FPC: 3.0.4
Platform: x86_64-Win64-win32/win64
OS version: Windows 10 Home 2009 x86_64
```

**Comment by alexx2000** [comment]
Drop from Explorer fixed (revision 9843).

**Comment by WhiteChairFromIkea** [comment]
>On my system drag&drop from Bitvise SFTP does not work with Explorer too (file with GUID created instead).
By saying "Explorer" do you mean "default MS Windows explorer.exe"? I am sure I didn't have any issues with explorer.exe (working only on windows 10 with latest updates). As can be seen from the video in OP, "GUID" string is replaced to user-friendly filename string after file is finished copying.

More info: there is no issues when drag-dropping from bitvise SSH to freecommander.exe (https://freecommander.com/en/downloads/) or explorer++ (https://github.com/derceg/explorerplusplus). 2nd is an open source project, so maybe parts of drop code can be borrowed from this project..?

**Comment by alexx2000** [comment]
Yes, MS Windows Explorer. Drag&drop from Bitvise SFTP does not work with all file managers on my system (Windows 10 21H1 x86_64). As I see Bitvise use some hackish method for drag&drop, it passing file in CF_HDROP clipboard format with GUID file name. So after drop, file with GUID name created and nothing else happen.

**Comment by Neustradamus** [comment]
@WhiteChairFromIkea: The official and good organization/repository:
- https://github.com/doublecmd/
- https://github.com/doublecmd/doublecmd

Please open at the good place.

Linked to:
- https://github.com/double-commander/doublecmd/issues/7

**Comment by codefolder** [comment]
On Windows with the latest update to the new Outlook, drag-and-drop works for emails dragged into Explorer, but Double Commander is ignored as target. Nothing (emails, files) can be dropped into DC from the new Outlook. Posting here as it may be a related issue.
